client/src/pages/Userprofile/UserWeatherSection.js

```diff
@@ -0,0 +1,141 @@
+import React, { useEffect, useState } from 'react';
+import { useGeolocation } from '../UserLocation/useGeolocation';
+
+export default function UserWeatherSection() {
+  const date = new Date();
+  const months = [
+    'January',
+    'February',
+    'March',
+    'April',
+    'May',
+    'June',
+    'July',
+    'August',
+    'September',
+    'October',
+    'November',
+    'December',
+  ];
+  const weekdays = [
+    'Sunday',
+    'Monday',
+    'Tuesday',
+    'Wednesday',
+    'Thursday',
+    'Friday',
+    'Saturday',
+  ];
+
+  const month = months[date.getMonth()];
+  const weekday = weekdays[date.getDay()];
+  const day = date.getDay();
+
+  const { data } = useGeolocation();
```

Comment:
There are a couple concerns about using `useGeolocation` here. 

https://github.com/waterthetrees/wtt_front/blob/64a4e567616f0c886e2a5f954f8ffc5287fc3d2f/client/src/pages/UserLocation/useGeolocation.js#L4-L5
1) The hook will ask for user's permission upon render of the component, since `useGeolocation` enables geolocation by default. This might be disruptive depending on how/where you add in `UserWeatherSection` (for example, if `UserWeatherSection` is called upon first load of the map)
2) By default, watching is set to true, which means that updates will continuously be sent to `data` here. This is a problem because `fetchWeather` is called whenever `data` is changed and will cause a ton of calls to `api.openweathermap`. 
For this, I suggest sending `useGeolocation({ watching: false })`

Reply:
resolved